## 1. Из колоды в 52 карты извлекаются случайным образом 4 карты.  

### a) Найти вероятность того, что все карты – крести.


$$
P = \frac{C^{4}_{13}}{C^{4}_{52}}
$$?

In [8]:
from math import factorial
def sochetanie (n,k):
 return factorial(n) // (factorial(k) * factorial(n-k))


In [24]:
def procent(a):
    return ("{:.2f}".format(a*100))


In [19]:
P = sochetanie(13,4) / sochetanie(52,4)
P

0.0026410564225690276

In [27]:
procent(P)

'0.26'

Вероятность 0,26 % что все 4 карты будут крести.

### б) Найти вероятность, что среди 4-х карт окажется хотя бы один туз.

$$
P = 1 - \frac{C^{4}_{48}}{C^{4}_{52}}
$$?

In [29]:
P = 1 - (sochetanie(48,4) / sochetanie(52,4))
P

0.2812632745405854

In [30]:
procent(P)

'28.13'

Вероятность 28.13 %, что среди 4 карт будет хоть один туз

## 2. На входной двери подъезда установлен кодовый замок, содержащий десять кнопок с цифрами от 0 до 9. Код содержит три цифры, которые нужно нажать одновременно. Какова вероятность того, что человек, не знающий код, откроет дверь с первой попытки?

$$
P = \frac{1}{C^{3}_{10}}
$$?

In [32]:
P = 1 / sochetanie(10,3)
procent(P)

'0.83'

Вероятность 0,83 % с первого раза открыть амок, не зная кода

## 3. В ящике имеется 15 деталей, из которых 9 окрашены. Рабочий случайным образом извлекает 3 детали. Какова вероятность того, что все извлеченные детали окрашены?

In [33]:
P = 9/15*8/14*7/13
procent(P)

'18.46'

Вероятность 18,46 % что все 3 детали буду окрашены.

## 4. В лотерее 100 билетов. Из них 2 выигрышных. Какова вероятность того, что 2 приобретенных билета окажутся выигрышными?

$$
P = \frac{1}{C^{2}_{100}}
$$?

In [35]:
P = 1/sochetanie(100,2)
procent(P)

'0.02'

Вероятность приобретения двух выигрышных билетов 0,02 %